spark/dl/src/main/scala/com/intel/analytics/bigdl/dataset/Sample.scala

```diff
@@ -440,37 +440,37 @@ private[bigdl] class SampleWithSparse[T: ClassTag](
   }
 }
 
-object SampleWithSparse {
+object TensorSample {
```

Comment:
just wondering do we have another Sample except for `TensorSample`?

Reply:
ArraySample